src/lib.rs

```diff
@@ -229,6 +236,23 @@ pub fn run_tests_generic(
     per_file_config: impl Fn(&mut Config, &Path, &[u8]) + Sync,
     status_emitter: impl StatusEmitter + Send,
 ) -> Result<()> {
+    // Nexttest emulation: we act as if we are one single test.
+    if configs.iter().any(|c| c.list) {
+        if configs.iter().any(|c| !c.run_only_ignored) {
+            println!("ui_test: test");
+        }
+        return Ok(());
+    }
+    for config in &mut configs {
+        if config.filter_exact
+            && config.filter_files.len() == 1
+            && config.filter_files[0] == "ui_test"
```

Comment:
If `TESTNAME` is used there could be some other filters in here since args extends the config's list

Reply:
I guess... filtering out just one specific name seems not great tho

Reply:
Yeah I wouldn't be too fussed if you ignored that